larray/util/misc.py

```diff
@@ -592,3 +593,11 @@ def index_by_id(seq, value):
         if item is value:
             return i
     raise ValueError("%s is not in list" % value)
+
+
+def renamed_to(old_name, newfunc, stacklevel=2):
+    def wrapper(*args, **kwargs):
```

Comment:
You do not use functools.wrap... I don't know whether it applies in this case, but if it does not, please remove the extra import.